packages/components/nodes/vectorstores/Redis_Existing/Redis_Existing.ts

```diff
@@ -0,0 +1,106 @@
+import { INode, INodeData, INodeOutputsValue, INodeParams } from '../../../src/Interface'
+import { Embeddings } from 'langchain/embeddings/base'
+import { getBaseClasses } from '../../../src/utils'
+import { createClient } from 'redis'
+import { RedisVectorStore, RedisVectorStoreConfig } from 'langchain/vectorstores/redis'
+
+class Redis_Existing_VectorStores implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+    outputs: INodeOutputsValue[]
+
+    constructor() {
+        this.label = 'Redis Load Existing Index'
+        this.name = 'redisExistingIndex'
+        this.type = 'Redis'
+        this.icon = 'redis.svg'
+        this.category = 'Vector Stores'
+        this.description = 'Load existing index from Redis (i.e: Document has been upserted)'
+        this.baseClasses = [this.type, 'VectorStoreRetriever', 'BaseRetriever']
+        this.inputs = [
+            {
+                label: 'Embeddings',
+                name: 'embeddings',
+                type: 'Embeddings'
+            },
+            {
+                label: 'Redis URI',
+                name: 'redisURI',
+                type: 'string'
+            },
+            {
+                label: 'Redis Index',
+                name: 'redisIndex',
+                type: 'string'
+            },
+            {
+                label: 'Top K',
+                name: 'topK',
+                description: 'Number of top results to fetch. Default to 4',
+                placeholder: '4',
+                type: 'number',
+                additionalParams: true,
+                optional: true
+            },
+            {
+                label: 'Redis Filters',
+                name: 'redisFilters',
+                type: 'json',
+                additionalParams: true,
```

Comment:
redisFilters is not `json` type. Although Flowise doesn't yet have `array` type where we can use to show in UI, for now we can use `type: string` and `rows: 4` to display a large textarea in UI. Then we have to put a placeholder to let user know what is the expected input.

So:
`placeholder: '['foo', 'bar']'`

and add a description:
```
description: `List of strings, applied using the OR operator in the metadata key. For example: filter = ['foo', 'bar'] => this will filter all metadata containing either 'foo' OR 'bar'`
```